lib/chef/application.rb

```diff
@@ -313,7 +314,7 @@ def fork_chef_client
             " finishing converge to exit normally (send SIGINT to terminate immediately)")
         end
 
-        client_solo = chef_config[:solo] ? "chef-solo" : "chef-client"
```

Comment:
we probably need a chef-solo variable as well

Reply:
I'll start a list to the proposal doc so we don't forget any.

Would you like for chef-solo be addressed in this same PR?

Reply:
It's not a blocker for this. It should just get done at some point